docs/actions.md

```diff
@@ -1,182 +1,117 @@
 # Actions
 
-Use [actions](/docs/api.md#actions) to update the state.
+Actions are functions which take the current [state](/docs/state.md) and return a partial state or a [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) that resolves to a partial state. Actions represent the intent to manipulate the state tree.
```

Comment:
> Actions represent the intent to manipulate the state tree.

I still disagree for apps that are more complex than a hello world. I very often use them to e.g. communicate via `socket.io`/`postMessage`/`http`/etc..